ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,375 @@
+| blockNumber       | QUANTITY        | Block number where this transaction was in.                                                |
+| cumulativeGasUsed | QUANTITY        | The total amount of gas used when this transaction was executed in the block.              |
+| effectiveGasPrice | QUANTITY        | The sum of the base fee and tip paid per unit of gas.                                      |
+| gasUsed           | QUANTITY        | The amount of gas used by this specific transaction alone.                                 |
+| contractAddress   | DATA, 20 Bytes  | The contract address created, if the transaction was a contract creation, otherwise `null` |
+| logs              | ARRAY           | Array of log objects, which this transaction'S EXECUTION FRAME generated.                  |
+| logsBloom         | DATA, 256 Bytes | Bloom filter for light clients to quickly retrieve related logs.                           |
+| type              | QUANTITY        | Integer of the transaction type                                                            |
+| status            | QUANTITY        | Either 1 (success) or 0 (failure) status of the execution frame                            |
+
+#### Create a new JSON-RPC API method `eth_executeRip7560Transaction`
+
+Executes the entire RIP-7560 transaction in memory without broadcasting it or including it in a block.
+Does not require the transaction to be properly signed, meaning it continues execution after either an account
+or a paymaster contract make a `sigFailAccount` or `sigFailPaymaster` call.
+If all frames execute successfully, simply returns the data returned by the top level frame of the execution phase.
+If any of the validation or execution frames revers, returns an error object containing the revert message.
+If the transaction validation fails for any reason other than the failed signature check,
+returns an error object containing the details of the validation failure.
+
+Parameters:
+
+1. OBJECT - The RIP-7560 transaction object (as defined in `eth_sendRip7560Transaction`).
+   The `signature` field is optional.
+2. QUANTITY | TAG - integer block number, or the string "latest", "earliest", "pending", "safe" or "finalized"
+
+Returns:
+
+DATA - The return value of the `sender` execution frame.
+
+Error:
+
+DATA - The revert data of the first reverted frame.
+CODE - The error code indicating the type of error, which may include the entity that caused the revert on-chain.
+MESSAGE - The human-readable error that may include a decoding of the `DATA` field if possible.
+
+#### Create a new JSON-RPC API method `eth_estimateRip7560TransactionGas`
+
+Performs a search for gas limit values required to make each of the frames of the RIP-7560 transaction execute
+successfully and without running out of gas.
+Note that for validation frames, only valid calls to an appropriate `AA_ENTRY_POINT` callback,
```

Comment:
So, `acceptAccount` and `acceptPaymaster` should not be considered a "success" for the gas estimation? I think this may be a **very** confusing and hard-to-implement requirement.
There is a possibility of a validation function, especially the paymaster but also some more exotic accounts, that does not differentiate between gas estimation and execution.

Reply:
acceptPaymaster/acceptAccount are always considered success.
sigFailedAccount/sigFailPaymaster are "conditional" success - they don't revert eth_execute, but are reported back. They are considered success for the purpose of estimateGas